android/app/build.gradle

```diff
@@ -8,13 +8,13 @@ plugins {
 
 dependencies {
   implementation platform('com.google.firebase:firebase-bom:33.10.0')
-  coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.0.4'
+  coreLibraryDesugaring 'com.android.tools:desugar_jdk_libs:2.1.5'
 }
 
 android {
     namespace = "de.gruene.wkapp"
     compileSdk = flutter.compileSdkVersion
-    ndkVersion = "25.1.8937393"
+    ndkVersion = "28.1.13356709"
```

Comment:
@volkramweber @NikoHadouken @Stift does anybody know whether this is alright? The ndk version is now higher than the [flutter 3.32.5 default version (27.0.12077973)](https://github.com/flutter/flutter/blob/df16c241b77267f932be6b6eed6655ed7600b0f1/packages/flutter_tools/gradle/src/main/kotlin/FlutterExtension.kt#L42) as well as the one recommended by [AGP 8.6.1 (26.1.10909125)](https://developer.android.com/build/releases/past-releases/agp-8-6-0-release-notes).

Reply:
This seems alright. The default version in flutter afaik is used when not specified otherwise. Else it should be similar to package compiled against the highest version. I'm just a bit confused as the maplibre-gl has a package 0.22 (https://pub.dev/packages/maplibre_gl), but in Github no release with release notes